UI/window-basic-main.cpp

```diff
@@ -4847,6 +4847,20 @@ void OBSBasic::changeEvent(QEvent *event)
 			if (previewEnabled)
 				EnablePreviewDisplay(true);
 		}
+#ifdef __APPLE__
+	} else if (event->type() == QEvent::StyleChange) {
+		const char *theme_name = config_get_string(
+			App()->GlobalConfig(), "General", "CurrentTheme3");
+		if (theme_name && strcmp(theme_name, "Yami") == 0) {
+			if (!fontsizePatched) {
+				fontsizePatched = true;
+				setStyleSheet(styleSheet().append(
+					QString("font-size: 13pt;")));
```

Comment:
This overrides every font-size in the app, which is an issue as not all font sizes are originally 10 (see the Yami.qss file for examples, like `10.5` in QDockWidget or `26` in the about dialog).

Reply:
That doesn't seem to be the case though - other font-sizes retain their size:

<img width="952" alt="Bildschirmfoto 2022-08-10 um 13 52 29" src="https://user-images.githubusercontent.com/9436503/183894465-717efe8c-c3c9-4720-9177-dc910006bee8.png">

Also this only sets the font-size on `OBSBasic`, more specific style sheet properties should still be applied regardless.

Reply:
This is how it currently looks, the "OBS Studio" and the version are supposed to be larger:
<img width="840" alt="image" src="https://user-images.githubusercontent.com/59806498/184043537-a3e345a0-43d1-4689-81b3-a540b9c85203.png">